assets/src/scripts/hydrograph/axes.js

```diff
@@ -39,13 +39,13 @@ function createAxes(xScale, yScale, yTickSize) {
 
 /**
  * Adds the given axes to a node
- * @param  {Selected} options.plot    Node to append to
- * @param  {D3 Axis} options.xAxis    x-axis
- * @param  {D3 Axis} options.yAxis    y-axis
- * @param  {Object} options.xLoc      {x, y} location of x-axis
- * @param  {Object} options.yLoc      {x, y} location of y-axis
- * @param  {Object} options.yLabelLoc {x, y} location of y-axis label
- * @param  {String} options.yTitle    y-axis label
+ * @param  {Object} plot      Node to append to
+ * @param  {Object} xAxis     D3 Axis x-axis
+ * @param  {Object} yAxis     D3 Axis y-axis
+ * @param  {Object} xLoc      {x, y} location of x-axis
+ * @param  {Object} yLoc      {x, y} location of y-axis
+ * @param  {Object} yLabelLoc {x, y} location of y-axis label
+ * @param  {String} yTitle    y-axis label
  */
```

Comment:
The options.xxx syntax indicates that the parameter is a single object, by convention called options. Here's the JSDoc syntax: http://usejsdoc.org/tags-param.html#parameters-with-properties